src/PKSim.Core/Snapshots/Mappers/OutputSchemaMapper.cs

```diff
@@ -10,10 +11,14 @@ namespace PKSim.Core.Snapshots.Mappers
    public class OutputSchemaMapper : SnapshotMapperBase<ModelOutputSchema, SnapshotOutputSchema>
    {
       private readonly OutputIntervalMapper _outputIntervalMapper;
+      private readonly IOutputSchemaFactory _outputSchemaFactory;
+      private readonly IContainerTask _containerTask;
 
-      public OutputSchemaMapper(OutputIntervalMapper outputIntervalMapper)
+      public OutputSchemaMapper(OutputIntervalMapper outputIntervalMapper, IOutputSchemaFactory outputSchemaFactory, IContainerTask _containerTask)
```

Comment:
Is this supposed to be _containerTask or containerTask?

Reply:
containerTask

Reply:
done